pkg/cmd/build/new.go

```diff
@@ -20,17 +22,21 @@ func NewCmdBuildNew(f *factory.Factory) *cobra.Command {
 
 	cmd := cobra.Command{
 		DisableFlagsInUseLine: true,
-		Use:                   "new <pipeline> [flags]",
+		Use:                   "new [pipeline] [flags]",
 		Short:                 "Creates a new pipeline build",
-		Args:                  cobra.ExactArgs(1),
+		Args:                  cobra.MaximumNArgs(1),
```

Comment:
I've changed this here/now because more resolvers are coming shortly which will make this argument optional. For now it is actually required though but it seemed like an okay thing to do